Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg

```diff
@@ -0,0 +1,71 @@
+PART
+{
+// --- general parameters ---
+name = kOSMachineRad
+module = Part
+author = Peter Goddard
+
+// --- asset parameters ---
+mesh = model/model.mu
+scale = 1
+rescaleFactor = 0.99999999999
+iconCenter = 0, 0, 0
+
+// --- node definitions ---
+node_attach = 0.0, 0.0, 0.0, 1, 0, 0
+
+// --- Tech tree ---
+TechRequired = flightControl
+
+// --- editor parameters ---
+cost = 2200
+entryCost = 4200
+category = Control
+subcategory = 0
+title = CompoMax Radial Tubeless
+manufacturer = Squalid-State Devices Inc.
+description = Would you trust life and limb to a mindless autopilot, powered by untested software you hastily wrote yourself? Spacefaring kerbals would!
+
+// attachment rules: stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+attachRules = 0,1,0,0,1
+
+// --- standard part parameters ---
+mass = 0.00001
+dragModelType = default
+maximum_drag = 0.0
+minimum_drag = 0.0
+angularDrag = 0
+crashTolerance = 6
+maxTemp = 3400
+
+
+MODULE
+{
+	name = kOSProcessor
+	diskSpace = 60000

```

Comment:
Done, as far as I’m concerned.

BTW I really enjoyed learning how to make parts. 
It’s really quite simple once you get the hang of it.


> On Mar 18, 2015, at 3:40 PM, Chris Woerz <notifications@github.com> wrote:
> 
> In Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg <https://github.com/KSP-KOS/KOS/pull/679#discussion_r26699329>:
> 
> > +attachRules = 0,1,0,0,1
> > +
> > +// --- standard part parameters ---
> > +mass = 0.00001
> > +dragModelType = default
> > +maximum_drag = 0.0
> > +minimum_drag = 0.0
> > +angularDrag = 0
> > +crashTolerance = 6
> > +maxTemp = 3400
> > +
> > +
> > +MODULE
> > +{
> > +	name = kOSProcessor
> > +	diskSpace = 60000
> So i guess what i am proposing is either
> 
> Move the part farther up the tech tree
> or
> 
> reduce the available space so people dont add it to their craft and run out of power right away
> if i was picking i would leave everything the same and just move it to to unmanned tech.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KSP-KOS/KOS/pull/679/files#r26699329>.
>